<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelScreen</class>
 <widget class="QDialog" name="LabelScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labels</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="screenHeader">
     <property name="text">
      <string>Label Objects</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLabel" name="sectionHeader_2">
     <property name="text">
      <string>Label Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="3" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QComboBox" name="labelTypeChoice">
     <item>
      <property name="text">
       <string>ruler</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>text</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="addLabelButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="2">
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item row="3" column="4" rowspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Color</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="6" colspan="3">
    <widget class="QListWidget" name="labelBrowser">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="ValueSlider" name="redSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="ValueSlider" name="greenSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="ValueSlider" name="blueSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="ValueSlider" name="textSizeSlider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="hideButton">
       <property name="text">
        <string>Hide</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="showButton">
       <property name="text">
        <string>Simple</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="highlightButton">
       <property name="text">
        <string>Full</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="4" rowspan="2" colspan="2">
    <widget class="QStackedWidget" name="pageStack">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="textUIGroup">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="sectionHeader">
         <property name="text">
          <string>Text Options</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Offset </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ValueSlider" name="textOffsetSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="fixedButton">
         <property name="text">
          <string>Fixed</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>X Pos</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ValueSlider" name="xPosSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Y Pos</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ValueSlider" name="yPosSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="attachTextButton">
         <property name="text">
          <string>Attach Text</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>attachTextButton</zorder>
      <zorder>label_12</zorder>
      <zorder>yPosSlider</zorder>
      <zorder>label_13</zorder>
      <zorder>label_10</zorder>
      <zorder>sectionHeader</zorder>
      <zorder>fixedButton</zorder>
      <zorder>textOffsetSlider</zorder>
      <zorder>xPosSlider</zorder>
     </widget>
     <widget class="QWidget" name="rulerUIGroup">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="sectionHeader_3">
         <property name="text">
          <string>Ruler Options</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Axis</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Precision </string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Units </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="unitsInput"/>
         </item>
         <item>
          <widget class="QToolButton" name="setAllUnitsButton">
           <property name="text">
            <string>set all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Offset </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ValueSlider" name="precisionSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ValueSlider" name="rulerOffsetSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="xLockButton">
           <property name="text">
            <string>X </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="yLockButton">
           <property name="text">
            <string>Y </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="zLockButton">
           <property name="text">
            <string>Z </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="attachRulerButton">
         <property name="text">
          <string>Attach Ruler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noGroup"/>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="selAllButton">
     <property name="text">
      <string>Sel All</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QPushButton" name="removeLabelButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>textSizeSlider</zorder>
  <zorder>label_3</zorder>
  <zorder>nameInput</zorder>
  <zorder>greenSlider</zorder>
  <zorder>sectionHeader_2</zorder>
  <zorder>redSlider</zorder>
  <zorder>label_2</zorder>
  <zorder>blueSlider</zorder>
  <zorder>pageStack</zorder>
  <zorder>label</zorder>
  <zorder>selAllButton</zorder>
  <zorder>addLabelButton</zorder>
  <zorder>labelBrowser</zorder>
  <zorder>removeLabelButton</zorder>
  <zorder>labelTypeChoice</zorder>
  <zorder>screenHeader</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueSlider</class>
   <extends>QSlider</extends>
   <header>ValueSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
